LICENSE.txt

```diff
@@ -1,4 +1,5 @@
 Copyright (c) 2014 Sean Doyle
+Copyright (c) 2015 Sean Doyle and thoughtbot, inc.
```

Comment:
I don't speak Licenses, but I would go with `Copyright (c) 2014 - 2015 Sean Doyle and thoughtbot, inc`